src/app/legacy/components/SkipLinkWrapper/index.jsx

```diff
@@ -1,10 +1,11 @@
-import React from 'react';
+/** @jsx jsx */
+import { jsx } from '@emotion/react';
 import { node, string, shape } from 'prop-types';
 import styled from '@emotion/styled';
 import detokenise from '#psammead/psammead-detokeniser/src';
-import { visuallyHiddenTextStyle } from '#psammead/psammead-visually-hidden-text/src';
 import { getSansBold } from '#psammead/psammead-styles/src/font-styles';
 import { GEL_BREVIER } from '#psammead/gel-foundations/src/typography';
+import { visuallyHiddenTextStyle } from '../../../components/VisuallyHiddenText/index.styles';
```

Comment:
I am actually wondering if the SkipLink component can just invoke the VisuallyHiddenText component, instead of importing the styles?

Reply:
There was a code comment somewhere that stated something along the lines of the `visuallyHiddenTextStyle` or similar was designed so that we can be selective or target the usage of it in other aspects of the codebase. Currently struggling to find this...but I'm pretty sure it explains why this component doesn't simply invoke the `VisuallyHiddenText` component in full

Reply:
As discussed we have now imported the `visuallyHiddenStyle` into https://github.com/bbc/simorgh/blob/dba69718ea29b4e189e33968e86fcab7779a7ea1/src/app/lib/styles.const.js for the timebeing